crates/astria-sequencer/src/bridge/init_bridge_account_action.rs

```diff
@@ -93,21 +92,28 @@ impl ActionHandler for InitBridgeAccountAction {
             "insufficient funds for bridge account initialization",
         );
 
-        state.put_bridge_account_rollup_id(from, &self.rollup_id);
+        // No need to add context as this method already reports sufficient context on error.
+        state.put_bridge_account_rollup_id(&from, self.rollup_id)?;
         state
-            .put_bridge_account_ibc_asset(from, &self.asset)
+            .put_bridge_account_ibc_asset(&from, &self.asset)
             .wrap_err("failed to put asset ID")?;
-        state.put_bridge_account_sudo_address(from, self.sudo_address.map_or(from, Address::bytes));
+        // No need to add context as this method already reports sufficient context on error.
+        state.put_bridge_account_sudo_address(
+            &from,
+            self.sudo_address.map_or(from, |address| *address.bytes()),
```

Comment:
This looked more elegant before. :) We could just add `as_bytes` to the `AddressBytes` trait and keep `bytes` as before.

Reply:
Fixed in [132bddc](https://github.com/astriaorg/astria/pull/1492/commits/132bddc7e5a700a5289c61dee5188628a929795b).